backend/server/routers/model.py

```diff
@@ -28,7 +28,7 @@ class CourseDetails(BaseModel):
     level: int
     description: str
     study_level: str
-    school: Optional[str]
+    school: str | None
```

Comment:
Does this actually need to be optional? Setting it to just `str` doesn't cause any mypy errors.

Reply:
I don't remember why, but it might have been a thing with how data comes in / out from the frontend and also our database. But also I could be wrong